cairo/ethereum/cancun/state.cairo

```diff
@@ -211,6 +211,13 @@ func get_storage{poseidon_ptr: PoseidonBuiltin*, state: State}(
     return value;
 }
 
+func destroy_account{poseidon_ptr: PoseidonBuiltin*, state: State}(address: Address) {
+    destroy_storage{poseidon_ptr=poseidon_ptr, state=state}(address);
```

Comment:
you should not need explicit binding of implicit args if they have the same name

```suggestion
    destroy_storage(address);
```